x/erc20/keeper/evm_log_handler_test.go

```diff
@@ -88,6 +89,30 @@ func (suite *KeeperTestSuite) TestSendToIbcHandler() {
 			func() {},
 			nil,
 		},
+		{
+			"denomination trace not found",
+			func() {
+				amount := sdk.NewInt(100)
+				suite.app.Erc20Keeper.SetContractForDenom(suite.ctx, validDenom, contract)
+				coin := sdk.NewCoin(validDenom, amount)
+				err := suite.MintCoins(sdk.AccAddress(contract.Bytes()), sdk.NewCoins(coin))
+				suite.Require().NoError(err)
+				suite.ctx.SetIsCheckTx(false)
```

Comment:
`CheckTx` and `TraceTx` are both `false` by default，why should it be setted explicitly?